### ISSUE TITLE
Segfault in running inference code in Pytorch 0.4 on Raspberry Pi 3B

### ISSUE THREAD

**Issue by Ashutosh1995** [amrit-das/Custom-Model-Training-PyTorch#4]
I am trying to run the inference code of ERFNet to generate the label masks but I am getting seg fault whenever I try to save the predictions. It randomly runs and then gives seg fault. There are total 254 images in the validation set.

Please suggest !!

**Comment by amrit-das** [comment]
Could you please also post the link to the code so that I can run and see the problem for myself.
